src/azul/chalice.py

```diff
@@ -141,6 +141,13 @@ def __init__(self,
         self.unit_test = unit_test
         self.non_interactive_routes: set[tuple[str, str]] = set()
         assert 'paths' not in spec, 'The top-level spec must not define paths'
+        # Inject contact details into both indexer and service app Swagger specs
```

Comment:
```suggestion
        # Inject contact details into both indexer and service app Swagger specs
```

The details are injected for *any* subclass. The explicit mention of the subclasses is a brittle and unnecessary detail.